relationships/candidates/MEMCACHEDNODE.yml

```diff
@@ -0,0 +1,18 @@
+category: MEMCACHEDNODE
+lookups:
+  - entityTypes:
+      - domain: INFRA
+        type: AWSELASTICACHEMEMCACHEDNODE
+    tags:
+      matchingMode: ALL
+      predicates:
+        - tagKeys: ["aws.elasticache.endpointAddress"]
+          field: endpoint
+        - tagKeys: ["aws.elasticache.endpointPort"]
+          field: port
+    onMatch:
+      onMultipleMatches: RELATE_ALL
+    onMiss:
+      action: CREATE_UNINSTRUMENTED
+      uninstrumented:
+        type: MEMCACHED
```

Comment:
Would `MEMCACHEDNODE` make sense ?
`MEMCACHED` seeems ambiguous to me

Reply:
Agree. I have added two uninstrumented types, one for cluster and one for node.